src/ott/neural/layers.py

```diff
@@ -81,59 +78,82 @@ def __call__(self, inputs: jnp.ndarray) -> jnp.ndarray:
 
 
 class PosDefPotentials(nn.Module):
-  r"""A layer to output :math:`\frac{1}{2} ||A_i^T (x - b_i)||^2_i` potentials.
+  """A layer to output  0.5 x^T(A_i A_i^T + Diag(d_i^2))x + b_i^T x + c_i potentials.
```

Comment:
do we really want to use a `d_i^2`? wouldn't it be easier to have a `rectifier(d_i)`? (e.g. relu, softplus etc?)